backend/src/api/routes/user.ts

```diff
@@ -0,0 +1,54 @@
+import type { Request, Response } from 'express';
+import { firebaseAdmin } from '../../admin';
+import type { UserToken } from '../../types/UserToken';
+import type { User } from '../../types/User';
+import { verifyUidToken } from '../../util/verifyUidToken';
+
+interface CreateUserParams extends User {
+  userToken: UserToken;
+}
+
+export async function createUser(request: Request, response: Response): Promise<void> {
+  const { displayName, email, uid, userToken, photoUrl }: Partial<CreateUserParams> = JSON.parse(
+    request.body
+  );
+  if (!displayName || !email || !uid || !userToken || !photoUrl) {
+    response.status(400).send({
+      error: {
+        code: 'error.missing_params',
+        message: 'Not all params are present',
+      },
+    });
+    return;
+  }
```

Comment:
If you turn `displayName, email, uid, userToken, photoUrl` into an array, we could turn a better error message: (pseudo code)
```suggestion
  const  keys: ReadyolyArray<keyof CreateUserParams> = ['displayName', 'email', 'uid', 'userToken', 'photoUrl']

  const params: Partial<CreateUserParams> = JSON.parse(
    request.body
  );
  if (keys.some(key => !params[key])) {
    response.status(400).send({
      error: {
        code: 'error.missing_params',
        message: `the following params are missing: ${keys.filter(key => !params[key].join(", ")}`,
      },
    });
    return;
  }
```